token-metadata/program/src/state.rs

```diff
@@ -587,6 +604,7 @@ impl ReservationListV2 {
 }
 
 #[repr(C)]
+#[cfg_attr(feature = "serde", derive(Serialize, Deserialize))]
```

Comment:
Why is the feature added here but not for `ReservationListV2`?

Reply:
And not for `ReservationListV1`?

Reply:
Also, `EditionMarker` is missing it.

Reply:
I might have missed it, but I added it to all the structs I needed to for it to compile, I'm happy to add it to them too

Reply:
Are you actually using the `Reservation` structs? 

`EditionMarker` definitely should have it.

Reply:
Since I was only using MetadataInstruction, I added only to the structs that the compiler complained didn't have Serde attr on them. But I'll add to these too

Reply:
It seems like ReservationListV2 has some serialize function already defined, so it rejects my addition of Serde, how do you want to handle that?

Reply:
I think you can skip all the `Reservation` structs and just add it to `EditionMarker` and we'll be good. Thanks for this PR!